### ISSUE TITLE
Breakpoints in feature files ignored

### ISSUE THREAD

**Issue by markdoth** [gasparnagy/SpecFlow.xUnitAdapter#9]
Just noticed that in feature files I've converted to remove code behind, Visual Studio ignores break points set in the feature file when debugging.

Still works on breakpoints set in the code in the step definitions

**Comment by gasparnagy** [comment]
@markdoth Yes, this is a known limitation. As far as I know, this can be solved by emitting a wrapper method call that uses the feature file position as the source reference, but I don't know the details yet.
